next.config.js

```diff
@@ -20,10 +20,11 @@ function interceptStdout(text) {
 // Intercept in dev and prod
 intercept(interceptStdout);
 
-const withPlugins = require('next-compose-plugins');
```

Comment:
I think we can avoid this import in other ways

Reply:
What's your opinion? 👀

Reply:
I think this version does it: https://github.com/Kwenta/kwenta/blob/stats/troy/next.config.js